### PR TITLE
mnit tileset and numbers

### PR DESCRIPTION
**Pull request by xymus**
Features from WBTW, used mainly to set the size of in-game hints and events boxes.

These modules are for Mnit but will be ported to Gamnit someday...

### CODE REVIEW

File: lib/mnit/tileset.nit [hunk 1]
```diff
@@ -84,6 +84,30 @@ class TileSetFont
 		if i == -1 then return null
 		return subimages[i]
 	end
+
+	# Distance between the beginning of a letter and the beginning of the next letter
```
Comment (privat) [inline lib/mnit/tileset.nit]: "of a letter **tile**  and the beginning of the next letter **tile**" My first reaction what that this included some kind of kerning, so I said "nice", then I read the code and was disappointed.

**Comment by privat** [comment]
+1

**Comment by xymus** [comment]
Rerolled to fix `advence` doc.

**Comment by Morriar** [comment]
+1

**Pull request merged**